dlt/common/configuration/providers/airflow.py

```diff
@@ -14,7 +14,6 @@ def name(self) -> str:
 
     def _look_vault(self, full_key: str, hint: type) -> str:
         """Get Airflow Variable with given `full_key`, return None if not found"""
-
```

Comment:
what is your black version? why this line is removed? did we have unformatted code (maybe)